content/en/docs/tutorials/acme/ingress.md

```diff
@@ -132,7 +132,7 @@ quickstart-nginx-ingress  0s
 ==> v1/Service
 quickstart-nginx-ingress-controller       0s
 quickstart-nginx-ingress-default-backend  0s
-
+```
```

Comment:
I think these back-ticks are meant to exist *after* the `NOTES` section, as the NOTES are output from the `helm install` command 😄

Reply:
Oh, I see it now! I had the impression that was not escaped.